docs/data_driven_testing.adoc

```diff
@@ -311,6 +311,35 @@ You can use named deconstruction with nested data pipes, but only on the innermo
 include::{sourcedir}/datadriven/DataSpec.groovy[tag=nested-and-named-multi-variable-data-pipe]
 ----
 
+== Cross-multiplying Data Providers
+
+Two or more consecutive data providers, be it a data table or a data pipe, can also be combined
+using a cartesian product using the `combined:` label between them. The following will result
+in these four executed tests
+
+- `a feature (1 3 5)`
+- `a feature (2 3 5)`
+- `a feature (1 4 6)`
+- `a feature (2 4 6)`
```

Comment:
I find the name here a bit missleading the `a feature` is not the variable `a` or the parameters are not separated.
How about:
`feature (a:1, b:3, c:5)`